week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,16 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function percentage(number) {
+  return Math.round(100 * number);
+}
+function massage(numberOfStudents, numberOfMentors) {
+  var total = numberOfStudents + numberOfMentors;
+  var percentageStudents = percentage(numberOfStudents / total);
+  var percentageMentors = percentage(numberOfMentors / total);
+
+  var massage1 = "Percentage Students = " + percentageStudents + "%";
+  var massage2 = "Percentage Mentors =" + percentageMentors + "%";
+
+  console.log(massage1);
+  console.log(massage2);
+}
+
+massage(26, 10);
```

Comment:
you did a very good job alisina. keep it up.

Reply:
Good work, Alisina.